utils.sh

```diff
@@ -0,0 +1,242 @@
+#!/bin/bash
+
+# Build DAP test container & start the cluster
+function createDAPTestEnvironment() {
```

Comment:
Let's keep this in the `test.sh` file as it's only used by that script.

Reply:
Moved into test.sh